client/blocks/editor-checkout-modal/style.scss

```diff
@@ -0,0 +1,24 @@
+.editor-checkout-modal {
+	position: fixed;
+	top: 0;
+	right: -100%;
+	z-index: 999999;
```

Comment:
Nit; there is a SASS mixin you can use for this kind of thing. :-)

Reply:
Oh really, I've been... blessed... with working with modals and popovers and other things that float around the screen, that SASS mixin might just save me. ;)